next/src/components/_global/caseStudiesComponent/caseStudiesComponent.module.scss

```diff
@@ -0,0 +1,170 @@
+.caseStudiesComponent {
+  display: grid;
+  gap: clamp(50px, calc(30vw/7.68), 50px);
+
+  header {
+    display: grid;
+    gap: 14px;
+    max-width: 689px;
+
+    .description {
+      padding-top: clamp(6px, calc(6vw/7.68), 20px);
+    }
+
+    .subheading {
+      max-width: 492px;
+    }
+  }
+
+  .items {
+    display: grid;
+    gap: 50px;
+
+    .item {
+      display: grid;
+      margin-top: -10px;
+      gap: 60px;
+
+      .projectAssumptions {
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+
+        .heading {
+          max-width: 492px;
+        }
+
+        .items {
+          display: flex;
+          flex-wrap: wrap;
+          gap: 50px;
+
+          .item {
+            display: grid;
+            grid-template-columns: auto 1fr;
+            align-items: center;
+            gap: 15px;
+            margin-top: unset;
+
+            .icon {
+              width: clamp(31px, calc(43vw/7.68), 52px);
+              height: auto;
+              display: block;
+            }
+
+            .description {
+              >p {
+                font-weight: 400;
+              }
+            }
+          }
+        }
+      }
+
+      .nameWrapper {
+        display: grid;
+
+        height: 120px;
+        width: 100%;
+        border-radius: 16px;
+        background: url("../../../../public/background-image.webp"), var(--gradient);
```

Comment:
This image could not be that large. Use SVG here.

Reply:
The image is 1200px x 1200px. Shall I still do this?